blockchain/state_migration.go

```diff
@@ -209,8 +209,7 @@ func (bc *BlockChain) migrateState(rootHash common.Hash) error {
 	checkElapsed := time.Since(startCheck)
 	logger.Info("State migration : Consistency check is done", "elapsed", checkElapsed)
 
-	bc.db.FinishStateMigration()
-	logger.Info("State migration : Finished")
+	logger.Info("completed state migration")
```

Comment:
How about merging two lines of logs instead of individual lines?

Reply:
I reflected. :)